OvmfPkg/OvmfPkgX64.dsc

```diff
@@ -41,6 +41,8 @@
   DEFINE DISABLE_MTRR_PROGRAMMING = TRUE
   DEFINE IOMMU_ENABLE            = FALSE
   DEFINE SETUP_PASSWORD_ENABLE   = TRUE
+  DEFINE SERIAL_TERMINAL         = TRUE
+  DEFINE SERIAL_REDIRECTION_DEFAULT = FALSE

```

Comment:
I don't think we need those. We can have it enabled at all times, because if there is a serial port, we always want it enabled I think.

Reply:
@macpijan real hardware does not build OVMF target so not relevant

Reply:
> real hardware does not build OVMF target so not relevant

@miczyg1 

I meant that we might need to expose such option for UefiPayloadPkg.
Removed it for OVMF as reuested. Now the serial redirection is always enabled by default on OFMF, so tests can be executed right after the build.

Reply:
SERIAL_TERMINAL is defined in UefiPayloadPkg. Also the default value of the redirection option is handled by PCD. The value of the PCD is passed to the build command by coreboot build system via `--pcd <pcdname>=<value>` so all your concerns are already addressed long time ago.

Reply:
https://github.com/Dasharo/coreboot/blob/dasharo/release/payloads/external/edk2/Makefile#L251

Reply:
Thanks @miczyg1 it in fact resolves my concerns. Anything more needed to close this issue?